sites/friday/src/manage-resources/_index.md

```diff
@@ -23,23 +23,25 @@ This flexibility allows you to optimize performance and costs.
 You can even scale horizontally if your applications are struggling with high load, or if you're expecting a traffic spike,
 by adding more instances for your applications and workers.
 
-{{< note >}}
+{{< note  title="Tip">}}
 
-When you first deploy your {{% vendor/name %}} project, and whenever you add a new application or service,
-you get notified that you need to configure the resources on your environment:
+To get the best guidance possible on how to configure resources on {{% vendor/name %}},
+use `{{% vendor/cli %}} push` instead of `git push`.
 
-```bash
-The push completed but resources must be configured before deployment can succeed.
-```
+{{< /note >}}
 
-This is because {{% vendor/name %}} doesn't know the exact amount of resources your project needs to run smoothly.
-Therefore, your app can only be successfully deployed after you've configured those resources
-through the {{% vendor/name %}} Console or CLI.
+## Default resources
 
-For better guidance on how to configure resources,
-use `{{% vendor/cli %}} push` instead of `git push`.
+When you first deploy your {{% vendor/name %}} project, and whenever you add a new application or service,
+{{% vendor/name %}} allocates the following default resources to every container:
 
-{{< /note >}}
+| Resource type               | Amount |
+| --------------------------- | ----------- |
+| CPU                         | 0.5 |
+| RAM                         | Depends on the [container profile](#advanced-container-profiles). |
+| Disk size                   | 512 MB |
```

Comment:
For the disk size, we should mention that it's only if the application or service requires a disk.